### PR TITLE
[Create news] Warning text has got incorrect size. #1199

### PR DESCRIPTION
**Pull request by romanovskyroma**
change size of text when resolution 320px
[bug1199](https://ita-social-projects.github.io/GreenCityClient/#/welcome)

### CODE REVIEW

**Review by skrebetsnataliia — COMMENTED** [review]
Please, use "ng lint" та "npm run stylelint" to check for style issues.
image

File: src/app/component/shared/components/drag-and-drop/drag-and-drop.component.scss [hunk 1]
```diff
@@ -1,6 +1,7 @@
 .dropzone {
   height: 231px;
-  width: 426px;
+  // width: 426px;
```
Comment (mr-pavlenko) [inline src/app/component/shared/components/drag-and-drop/drag-and-drop.component.scss:3]: comment

File: src/app/component/eco-news/components/create-news/create-news.component.scss [hunk 2]
```diff
@@ -58,10 +58,15 @@ form {
...
+.container {
+  max-width: 1170px;
+  margin: 0 auto;
+}
+
 .wrapper {
   display: grid;
-  grid-template-columns: auto 430px;
-  grid-column-gap: 30px;
+  grid-template-columns: auto 456px;
```
Comment (mr-pavlenko) [inline src/app/component/eco-news/components/create-news/create-news.component.scss:68]: why did you use 456?

File: src/app/component/shared/components/drag-and-drop/drag-and-drop.component.scss (hunk 1 above)
Comment (romanovskyroma) [reply src/app/component/shared/components/drag-and-drop/drag-and-drop.component.scss:3]: I delete comment,
and add width: 100%

File: src/app/component/eco-news/components/create-news/create-news.component.scss (hunk 2 above)
Comment (romanovskyroma) [reply src/app/component/eco-news/components/create-news/create-news.component.scss:68]: according to the mockup
create-news(456x231)

**Comment by romanovskyroma** [comment]
> Please, use "ng lint" and "npm run stylelint" to check for style issues.
> image

I fixed

**Review by luchward — APPROVED** [review]

**Review by skrebetsnataliia — APPROVED** [review]

**Review by mr-pavlenko — APPROVED** [review]

**Pull request merged**